src/sqs/branch.ts

```diff
@@ -2,8 +2,9 @@ import { WebhookPayloadCreate } from "@octokit/webhooks";
 import { Context, MessageHandler } from "./sqs";
 import { processBranch } from "../github/branch";
 import { createInstallationClient } from "~/src/util/get-github-client-config";
+import { GitHubAppConfigPayload } from "./github-app-config-payload";
 
-export type BranchMessagePayload = {
+export type BranchMessagePayload = GitHubAppConfigPayload & {
```

Comment:
Wouldn't it be simpler to just do 
```
{
   githubAppConfig: GitHubAppConfig;
   ...
}
```

Reply:
Will it be risking some payload use `githubAppConfig` while some use `gitHubAppConfig` as the field name?

Reply:
if you're concerned about that, you can create a `MessagePayload` parent interface, but we do need the type of the config itself as it will be used in other places.

Reply:
Fixed, extracted in single file.